.pre-commit-config.yaml

```diff
@@ -0,0 +1,16 @@
+repos:
+- repo: https://github.com/pre-commit/pre-commit-hooks
+  rev: v2.3.0
+  hooks:
+  - id: trailing-whitespace
+  - id: check-yaml
+  - id: end-of-file-fixer
+  - id: requirements-txt-fixer
```

Comment:
You should include the following pre commit hooks:
1. black (To format the code properly)
2. trailing whitespace (To trim the extra spaces)
3. isort (To sort the imports)

Remove all the hooks that are not mentioned in the above points.

Reply:
I've added above mentioned hooks.